identity-apps-core/apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -1033,7 +1033,9 @@
                     <% }
                     String clientId = request.getParameter("client_id");
                     String urlParameters = "";
-                    if ((StringUtils.equals("CONSOLE",clientId)
+                    if (!isSelfSignUpEnabledInTenant
+                            && StringUtils.isNotBlank(application.getInitParameter("AccountRegisterEndpointURL"))
+                            && (StringUtils.equals("CONSOLE",clientId)
                             || (StringUtils.equals("MY_ACCOUNT",clientId)
```

Comment:
This will hide the self signup URL for business apps, won't that be issue?

Reply:
In business apps, button will be displayed as similar to asgardeo. Only super tenant users will see the link if the `registartion_url` is setup in configuration and self sign up is not enabled.